gluoncv/data/mhp.py

```diff
@@ -0,0 +1,182 @@
+"""Multi-Human-Parsing V1 Dataset."""
+import os
+import numpy as np
+from PIL import Image
+from PIL import ImageFile
+
+import mxnet as mx
+from .segbase import SegmentationDataset
+
+ImageFile.LOAD_TRUNCATED_IMAGES = True
+
+
+class MHPV1Segmentation(SegmentationDataset):
+    """Multi-Human-Parsing V1 Dataset.
+    Parameters
+    ----------
+    root : string
+        Path to VOCdevkit folder. Default is '$(HOME)/mxnet/datasplits/mhp/LV-MHP-v1'
```

Comment:
Still VOCdevkit? Please change it accordingly.

Reply:
Also, the default folder to put all datasets is `(HOME)/.mxnet/datasets/`, not `(HOME)/mxnet/datasplits/`.